encoding/resampling.md

```diff
@@ -1,35 +1,429 @@
+as it produces only minimal artifacting,
+at the cost of slight blurriness.
+
+VapourSynth example:
+
+```py
+clip = core.resize.Bicubic(src, w, h, filter_param_a=0, filter_param_b=0.5) # “filter_param_a” and “filter_param_b” refer to B and C, respectively
+```
+
+##### Lanczos
+
+Lanczos is generally considered
+a very high-quality resampler for upscaling,
+especially its [EWA version][#ewa-elliptical-weighted-averaging-also-cylindrical-polar-circular].
+
+It is usually slightly sharper than Mitchell (Bicubic b=c=1/3),
+but might produce slightly more ringing.
+
+Lanczos takes an additional parameter
+that controls the filter’s number of lobes,
+also called taps.
+Increasing the number of lobes
+improves sharpness at the cost of increased ringing.
+You might occasionally see the tap count
+appended to the filter name to clarify
+the exact filter used, e.g. Lanczos2 for 2 taps.
+
+For downscaling,
+higher tap counts might help in
+suppressing [Moiré effects][Moire].
+
+[Moire]: http://www.imagemagick.org/Usage/filter/#aliasing
+
+```py
+clip = core.resize.Lanczos(src, w, h, filter_param_a=2) # “filter_param_a” specifies the tap count
+```
+
+##### Spline
+
+Spline is another high-quality resizer.
+
+Spline, like Lanczos,
+can be finetuned by configuring its number of lobes.
+Unlike Lanczos, 
+however,
+Splines with different tap counts are usually split
+into seperate functions,
+with ``(tap_count*2)**2`` appended to their name,
+e.g. Spline36 for 3 taps, Spline64 for 4, etc.
+(This number repesents the total amount of input pixels
+involved in the calculation of any given output pixel.)
+
+Spline36 is a very popular choice for downscaling,
+since it is fairly artifact-free yet decently sharp.
+For upscaling,
+it looks similiar to Lanczos3,
+though arguably slightly less artifacted.
+
+VS example:
+
+```py
+clip = core.resize.Spline36(src, w, h)
+```
+
+Higher tap counts can be used via fmtconv:
+
+```py
+clip = core.fmtc.resample(src, w, h, kernel="spline", taps=6) # equivalent to Spline144
+```
+
+##### Gauss
+
+The Gaussian filter is very special in that its [Fourier transform][^1]
+is another Gaussian
+whose width is antiproportional to the spatial function’s.
+This can be harnessed to remove and amplify high frequencies 
+in a very controllable way.
+Widening the filter,
+for example,
+will confine the output to small frequencies (blurrier),
+whereas squashing it will amplify higher frequencies (more aliasing).
+
+In practice,
+though,
+the Gaussian filter isn’t all too useful for regular resizing.
+However,
+it can be used to accurately
+emulate a Gaussian blur (when used without resizing) in VapourSynth.
+
+For example:
+
+```py
+blurred = core.fmtc.resample(src, kernel="gauss", fh=-1, fv=-1, a1=1)
+```
+
+``fh=-1, fv=-1`` forces the processing when no resizing is performed.
+``a1`` controls the blurring:
+the higher,
+the sharper the image.
+
+
+#### Interpolation filters
```

Comment:
Following the above, this should be a h3 if inteded to be on the same level as "Filters", although I can see this being a subheading of that as well.